package.json

```diff
@@ -70,19 +70,19 @@
     "validator": "10.11.0"
   },
   "peerDependencies": {
-    "dotenv": "^8.2.0",
-    "dotenv-webpack": "^3.0.0",
-    "webpack": "^4.44.2",
-    "webpack-merge": "^5.4.0",
-    "mini-css-extract-plugin": "^0.11.2",
-    "html-webpack-new-relic-plugin": "^2.0.0",
+    "dotenv": "8.2.0",
+    "dotenv-webpack": "3.0.0",
+    "webpack": "4.44.2",
+    "webpack-merge": "5.4.0",
+    "mini-css-extract-plugin": "0.11.2",
+    "html-webpack-new-relic-plugin": "2.0.0",
     "html-webpack-plugin": "4.5.0",
-    "clean-webpack-plugin": "^3.0.0"
+    "clean-webpack-plugin": "3.0.0"
```

Comment:
Note: these are only peerDeps to avoid linting errors in our custom webpack configs. These dependencies are installed via the edx/frontend-build `devDependency`.